lib/modem_info/modem_info.c

```diff
@@ -895,6 +896,32 @@ int modem_info_get_rsrp(int *val)
 	return 0;
 }
 
+int modem_info_get_rsrq(float *val)
+{
+	int ret;
+	int rsrq_idx;
+
+	if (val == NULL) {
+		return -EINVAL;
+	}
+
+	ret = nrf_modem_at_scanf("AT+CESQ",
+				 "+CESQ: %*d,%*d,%*d,%*d,%d,%*d", &rsrq_idx);
+
+	if (ret != 1) {
+		LOG_ERR("at_scanf_int failed");
+		return map_nrf_modem_at_scanf_error(ret);
+	}
+
+	if (rsrq_idx == CELL_RSRQ_INVALID) {
+		LOG_WRN("No valid RSRQ");
```

Comment:
I'm wondering if this warning should be removed. After all, this is normal behavior when the modem has no cell, so the warning log could be confusing.